docker-alpine/Dockerfile

```diff
@@ -22,16 +22,19 @@ VOLUME ["/sys/fs/cgroup"]
 # We need curl to get root certificates
 RUN apk update \
     && apk upgrade \
-    && apk --no-cache add mosquitto curl collectd sudo openrc
-
-# Create /run/lock
-RUN mkdir /run/lock
-RUN chmod 1777 /run/lock/
+    && apk --no-cache add mosquitto curl collectd collectd-mqtt sudo openrc
 
 # Add user and groups
 RUN addgroup -S tedge
 RUN adduser -g "" -H -D tedge -G tedge
 
+# Create useful directories
+RUN mkdir /run/lock && \
+    mkdir /var/tedge && \
+    mkdir /etc/collectd/collectd.conf.d && \
+    chmod 1777 /run/lock/ && \
+    chown tedge:tedge /var/tedge
```

Comment:
Also, `RUN sudo -u tedge tedge_agent --init` creates `/var/tedge` by the owner `tedge:tedge`. This line can be also omitted.